search/constants.py

```diff
@@ -0,0 +1,17 @@
+"""
+Search application settings and constants that can be accessed globally.
+"""
+from typing import List
+
+# Add settings and constants here:
+
+REQUIRED_SETTINGS: List[str] = [
```

Comment:
So not resupplying this disables the check?

Reply:
If this is not supplied then that would raise an exception causing the app to crash.

Reply:
And supplying an empty list?

Seems like constants and settings are being confused.

Reply:
What would you suggest?

Reply:
Move settings to something called `settings` and/or move this to `check_required_settings`?

Reply:
updated.